content/07.supplemental.md

```diff
@@ -1,35 +1,14 @@
-Figure created with [BioRender.com](BioRender.com).
-
-**Figure S2. Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.**  
-Correlation (A) and violin (B) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, Mutect2, and VarDict) used for PBTA samples. 
-Upset plot (C) showing overlap of variant calls. Correlation (D) and violin (E) plots of mutation variant allele frequencies (VAFs) comparing the variant callers (Lancet, Strelka2, and Mutect2) used for TCGA samples. 
-Upset plot (F) showing overlap of variant calls.
-Violin plots (G) showing VAFs for Lancet calls performed on WGS and WXS from the same tumor (N = 52 samples from 13 patients).
-Cumulative distribution TMB plots for PBTA (H) and TCGA (I) tumors using consensus SNV calls.
-
-**Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.**
-(A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N < 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3). 
-Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. 
-Only primary tumors with mutations in the listed genes are shown. 
-Multiple CNVs are denoted as a complex event. 
-(B) Genome-wide plot of CNV alterations by broad histology. 
-Each row represents one sample. 
-Box and whisker plots of number of CNV breaks (C) or SV breaks (D) by number of chromothripsis regions. 
-
-**Figure S4. Related to Figure 3.** 
-(A) Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. 
-(B) Proportion of Signature 1 plotted by phase of therapy for each cancer group.
-
-**Figure S5. Related to Figure 4**.
-(A) Receiver Operating Characteristic for _TP53_ classifier run on FPKM of poly-A RNA-Seq samples. 
-Correlation plots for telomerase scores (EXTEND) with RNA expression of _TERT_ (B) and _TERC_ (C).First two dimensions from UMAP of sample transcriptome data with points colored by `molecular_subtype` for medulloblastoma (D), ependymoma (E), low-grade glioma (F), and high-grade diffuse astrocytic tumors (G). 
+![**OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC. Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with [BioRender.com](BioRender.com).](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0c4a7e10bab352d7d1b8840d1559f68bdb46bf66/figures/pngs/figureS1.png?sanitize=true){#fig:S1 tag="S1" width="7in"}
```

Comment:
Out of curiosity, why are there different commits being used?

Reply:
Do you mean instead of no commit? The docs said that if outside of this repo, to use a versioned figure, so I pull each latest commit as the version. That alone didn't work even though the instructions suggested it did. What worked was to add the raw content piece as you suggested, even though the docs suggest that's svg-specific.

Reply:
The _latest commit_ in the analysis repository is `489d729cde4318654a26b0a9ef0dee6d706352cd`. The URLs for these figures don't use the same commit hash. 

This one uses: `0c4a7e10bab352d7d1b8840d1559f68bdb46bf66` 
S3 uses: `b78c63b52fe252ab26b3141272031d4c35504f6e`

Is there a reason for that?

Reply:
Ok, I was navigating to the wrong area, fixed in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/244/commits/1f716721374972b98944911c71ebfa8bddbc7d8f